package.json

```diff
@@ -31,7 +31,7 @@
     "tsconfig-paths": "^4.2.0",
     "ttab": "^0.8.0",
     "turbo": "^1.10.16",
-    "typescript": "^4.9.4"
+    "typescript": "^5.0.4"
```

Comment:
Nice!
It looks like typescript 5 has some nice new features.
Especially it looks like we can now remove the `experimentalDecorators` flag from some `tsconfig.json`

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-0.html#differences-with-experimental-legacy-decorators

capture: https://github.com/anonklub/anonklub/issues/300

Reply:
Well also I had to upgrade it since viem works with v5.

Reply:
Yes, I think that is also why we were having some eslint issues.
Normally view gives you the types from the ABI